create_graphics.py

```diff
@@ -546,26 +552,36 @@ def graphics_driver(cla):
             print()
             print((('-' * 80)+'\n') * 2)
 
-            if cla.graphic_type == 'skewts':
-                create_skewt(cla, fhr, grib_path, workdir)
-            else:
-                gribfiles = gather_gribfiles(cla, fhr, gribfiles)
-                create_maps(cla,
-                            fhr=fhr,
-                            gribfiles=gribfiles,
-                            workdir=workdir,
-                            )
-
```

Comment:
This section needs to stay as-is so that the graphics are created no matter whether you intend to zip the files.